### PR TITLE
Updated age calculator to be a first class citizent

### PR DESCRIPTION
**Pull request by brianz**
Why
---

- Get the age calculator example working really well
- Update the README and other supporting docs to be 100% accurate

This change addresses the need by
---------------------------------

- Updating code
- Adding lots of tests
- Testing on a real device
- Removed some duplicate code between the lazysusan `Request` object and `helpers`

Next Steps
----------

- Could use some more tests
- Update the Makefile pip install once this branch is merged

### CODE REVIEW

File: lazysusan/helpers.py [hunk 1]
```diff
@@ -1,10 +1,3 @@
-def get_slot_value(request, slot_name):
-    try:
-        return request["intent"]["slots"][slot_name]["value"]
-    except KeyError:
-        return None
-
-
 def build_response(state_key, message, state_machine):
     response = state_machine[state_key]["response"]
     response["outputSpeech"]["text"] = message
```
Comment (samcdavid) [inline lazysusan/helpers.py:3]: This whole file should probably go away. This should be built into lazysusan now.

Comment (brianz) [reply lazysusan/helpers.py:3]: I agree.  Perhaps we can do that in an upcoming change?

Comment (samcdavid) [reply lazysusan/helpers.py:3]: Sounds good to me.

**Review by samcdavid — APPROVED** [review]
Code climate issues are with the examples. Since they are not part of the core logic of the framework I'm not particularly concerned with them.

**Pull request merged**